contracts/src/v0.8/automation/dev/v2_2/AutomationRegistry2_2.sol

```diff
@@ -133,8 +141,9 @@ contract AutomationRegistry2_2 is AutomationRegistryBase2_2, OCR2Abstract, Chain
     // This is the overall gas overhead that will be split across performed upkeeps
     // Take upper bound of 16 gas per callData bytes, which is approximated to be reportLength
     // Rest of msg.data is accounted for in accounting overheads
+    // NOTE in process of changing acounting, so pre-emptively changed reportLength to msg.data.length
     gasOverhead =
-      (gasOverhead - gasleft() + 16 * rawReport.length) +
+      (gasOverhead - gasleft() + 16 * msg.data.length) +
```

Comment:
just out of curiousity, msg.data and rawReport are equivalent?

Reply:
you can see the comment I added above